src/router/metadata/cap.rest.security.template.ts

```diff
@@ -8,7 +8,7 @@ import { Auth } from 'fhir-works-on-aws-interface';
 export default function makeSecurity(authConfig: Auth) {
     if (authConfig.strategy.service) {
         let security = {
-            cors: false,
+            cors: true,

```

Comment:
This should be a variable passed into this function; see: https://github.com/awslabs/fhir-works-on-aws-routing/pull/18/files

Reply:
Hi Robert,
I think this needs to be a constant - once we have the Cors middleware package in the router, cors headers will always be available in requests served the router.  With no config, the middleware will provide a basic setup; adding a CorsOptions will allow users to customize the settings if required.
A real-world FHIR server will need to provide cors headers so I don't think there is much value in making this configurable.  If it was configurable we'd also need to dynamically switch in the middleware according to this setting.

Reply:
Yeah totally understand your point here. One thing to note is that the capability statement is generated dynamically for every `/metadata` invocation based on the `config`.  Here is the metadata router: https://github.com/awslabs/fhir-works-on-aws-routing/blob/d120da1c0c3502c19f8da014335c95c840a787e2/src/router/routes/metadataRoute.ts#L29

So having the `isCorsEnabled` variable be passed in aligns with passing in of the `corsOptions` object

We want to be able to have CORS and no CORS - unless somewhere in the FHIR spec specifies CORS is required

Reply:
The FHIR spec says that CORS _should_ be implemented although it's not compulsory:
https://www.hl7.org/fhir/http.html#3.1.0.1.4
I can't think of a situation where you wouldn't want it available.

Reply:
Good find on the link - when looking through it I found: 

> Note: to support browser-based client applications, servers SHOULD implement cross-origin resource sharing (CORS)  for the interactions documented here.

So I guess one use-case is if the FHIR server doesn't have a need to support browser-based client applications, then setting up CORS is not needed